Sources/NostrSDK/EventCreating.swift

```diff
@@ -612,13 +612,11 @@ public extension EventCreating {
         var tags: [Tag] = [
             calendarEventCoordinates.tag,
             Tag(name: .identifier, value: identifier),
-            Tag(name: .labelNamespace, value: "status"),
-            Tag(name: .label, value: status.rawValue, otherParameters: ["status"])
+            Tag(name: "status", value: status.rawValue)
```

Comment:
I chose not to standardize the `status` and `fb` tags in the `TagName` enum as their usages are constrained to only calendar event RSVPs and it seems that we want kind-specific tag enums somehow if that's possible, if we figure that out down the line.